### ISSUE TITLE
Add mongo as dependancie in the chart

### ISSUE THREAD

**Issue by Tchoupinax** [suda/ackee-chart#1]
Hello,

With helm we can declare dependancy. I would love that this chart automatically install a mongoDB database (can be configuring in values.yaml of course). [https://helm.sh/docs/helm/helm_dependency/]

I can make the pull request if you are interested :)

Thank you for your work !

**Comment by suda** [comment]
Hi! Yes, it is possible to declare a dependency but there are many ways of deploying MongoDB with Helm. There's the default [Bitnami](https://bitnami.com/stack/mongodb/helm) chart, [Sharded MongoDB](https://bitnami.com/stack/mongodb-sharded/helm) one and you even can use [kubedb](https://kubedb.com/docs/v0.13.0-rc.0/guides/mongodb/) operator to install it. It's also possible to just use an external, hosted MongoDB. I didn't want to make decisions for anyone, especially that everyone can have different needs. What if, instead of depending on a particular MongoDB, we update the README to tell the users how to quickly deploy it as well?

**Comment by Tchoupinax** [comment]
I agree, paragraph in the readme with one way to quickly deploy a MongoDB instance is a good thing.
Do you want i suggest it ?

**Comment by balibebas** [comment]
Those who want to preview Ackee locally using K3D or KiND would find value in a MongoDB integration. It doesn't have to be released with the software. You could add an examples directory and create some common integrations to help reduce friction for those evaluating the software.

**Comment by ghost** [comment]
Those who want to preview Ackee locally using K3D or KiND would find value in a MongoDB integration. It doesn't have to be released with the software. You could add an examples directory and create some common integrations to help reduce friction for those evaluating the software.

Here's how Gitea does it:
https://gitea.com/gitea/helm-chart/src/commit/94ee162ad57b7fe43ec3934b19431784cb796ff7/values.yaml#L109-L116

Note users can turn on and off the DB so they can use their own external database. This is a familiar pattern with Helm charts.

**Comment by suda** [comment]
The Gitea approach might work! Anyone willing to create a PR with this? :)
